src/components/GraphClient/queries/getMetadata.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Maybe rename this file getRecordMetadata.json and put it into queries/editor/

Reply:
This could be deleted if it's not needed as discussed below.